### PR TITLE
small tweaks to main page

### PR DESCRIPTION
**Pull request by Ofer-Idan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,8 +44,8 @@ <h1>Red Sky Ops</h1>
   <div class="content main-content">
     <div class="main card">
       <p>
-        Red Sky Ops is an AIOps platform for deploying, scaling and managing containerized applications in Kubernetes environments. It makes it easy for DevOps teams to manage the millions of possible combinations of application variables and configuration settings they're confronted with non-stop. With Red Sky Ops, they can automatically identify and implement the best settings for each application in any cloud environment.
-        Red Sky Ops handles this complexity to optimize application performance and lower infrastructure costs while dramatically cutting alert noise.
+        Red Sky Ops is an AIOps platform for deploying and optimizing containerized applications in Kubernetes environments. It makes it easy for DevOps teams to manage the millions of possible combinations of application variables they're confronted when deploying applications. With Red Sky Ops, they can identify and implement the best settings for each application in any cloud environment.
```
Comment (robpacheco) [inline index.html]: 'in *a* Kubernetes env..'

File: index.html [hunk 2]
```diff
@@ -44,8 +44,8 @@ <h1>Red Sky Ops</h1>
...
+        Red Sky Ops allows teams to streamline their application tuning process and have a centralized, organized view of of tjeir tuning results.
```
Comment (robpacheco) [inline index.html]: 'theirs' has a typo

File: index.html (hunk 1 above)
Comment (Ofer-Idan) [reply index.html]: It's "in Kubernetes environments"? you'd prefer it singular?

File: index.html (hunk 2 above)
Comment (Ofer-Idan) [reply index.html]: fixed, thanks.

**Pull request merged**